### PR TITLE
rename prefix to 'todo'

### PR DESCRIPTION
**Pull request by d3ns0n**

### CODE REVIEW

File: .angular-cli.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
       "test": "test.ts",
       "tsconfig": "tsconfig.app.json",
       "testTsconfig": "tsconfig.spec.json",
-      "prefix": "app",
+      "prefix": "todo",
```
Comment (wassertim) [inline .angular-cli.json]: with that you fix the ts-lint errors?

File: e2e/app.e2e-spec.ts [hunk 2]
```diff
@@ -9,6 +9,6 @@ describe('ang4-seo App', () => {
 
   it('should display welcome message', () => {
     page.navigateTo();
-    expect(page.getParagraphText()).toEqual('Welcome to app!');
+    expect(page.getParagraphText()).toEqual('Javascript Days');
```
Comment (wassertim) [inline e2e/app.e2e-spec.ts]: nice

File: src/app/auth/login/login.component.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
-import { Router } from '@angular/router';
-import { AuthService } from '../auth.service';
-import { User } from '../user';
-import { Subscription } from 'rxjs/Subscription';
+import {Component, OnDestroy, OnInit} from '@angular/core';
```
Comment (wassertim) [inline src/app/auth/login/login.component.ts]: not consistent

File: .angular-cli.json (hunk 1 above)
Comment (d3ns0n) [reply .angular-cli.json]: sadly not.

**Comment by d3ns0n** [comment]
to fix tslint errors you have to change the prefix in tslint.json. See 0c192ea

File: tslint.json [hunk 4]
```diff
@@ -117,13 +117,13 @@
...
       "attribute",
-      "app",
+      "todo",
       "camelCase"
     ],
     "component-selector": [
       true,
       "element",
-      "app",
+      "todo",
```
Comment (wassertim) [inline tslint.json]: nice 👍

**Pull request merged**